active_directory/datadog_checks/active_directory/data/conf.yaml.example

```diff
@@ -3,6 +3,8 @@ init_config:
 instances:
   # "." means the current host
   - host: .
+  # username: my_username
```

Comment:
Maybe a simple comment letting users know that these options are used to authenticate the connection to the host would look good.